python/sglang/srt/server_args.py

```diff
@@ -194,13 +194,6 @@ class ServerArgs:
     disaggregation_bootstrap_port: int = 8998
 
     def __post_init__(self):
-        # Expert parallelism
```

Comment:
please delete the later one. We need this one for some weight loading logics

Reply:
Sure